src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/.template.config.src/template.json

```diff
@@ -5,36 +5,33 @@
     "Blazor",
     "Hosted"
   ],
-  "description": "A project template for creating a Blazor application that runs on WebAssembly and is hosted on an ASP.NET Core server.",
-  "groupIdentity": "Microsoft.Web.Blazor.Hosted",
+  "description": "A project template for creating a Blazor app that runs on WebAssembly. This template can be used for web apps with rich dynamic user interfaces (UIs).",
+  "groupIdentity": "Microsoft.Web.Blazor.Wasm",
   "guids": [
     "4C26868E-5E7C-458D-82E3-040509D0C71F",
     "5990939C-7E7B-4CFA-86FF-44CA5756498A",
     "650B3CE7-2E93-4CC4-9F46-466686815EAA",
     "0AFFA7FD-4E37-4636-AB91-3753E746DB98"
   ],
-  "identity": "Microsoft.Web.Blazor.Hosted.CSharp",
-  "name": "Blazor (ASP.NET Core hosted)",
+  "identity": "Microsoft.Web.Blazor.Wasm.CSharp",
+  "name": "Blazor WebAssembly App",
   "preferNameDirectory": true,
   "primaryOutputs": [
+
     {
-      "condition": "(HostIdentifier == \"dotnetcli\" || HostIdentifier == \"dotnetcli-preview\")",
```

Comment:
These conditions are necessary to ensure the template works property in VS.

Reply:
I will add back